tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/AnalyzerTests.cs

```diff
@@ -28,10 +28,10 @@ public AnalyzerTests(ITestOutputHelper output)
                 "UA0001",
                 new[]
                 {
-                    new ExpectedDiagnostic("UA0001", new TextSpan(15, 17)),
-                    new ExpectedDiagnostic("UA0001", new TextSpan(34, 23)),
-                    new ExpectedDiagnostic("UA0001", new TextSpan(59, 37)),
-                    new ExpectedDiagnostic("UA0001", new TextSpan(184, 11))
+                    new ExpectedDiagnostic("UA0001", new TextSpan(15, 17), new TextSpan(14, 17)),
+                    new ExpectedDiagnostic("UA0001", new TextSpan(34, 23), new TextSpan(32, 23)),
```

Comment:
Is there no way to calculate the span for MacOS? This already looks like a nightmare to maintain.

Reply:
Not really because it all depends on how many lines the matched text spans.